stories/core/ExpandableBlock.stories.tsx

```diff
@@ -111,6 +111,7 @@ export const Accordion: Story<Partial<ExpandableBlockProps>> = (args) => {
 Accordion.argTypes = {
   title: { control: { disable: true } },
   isExpanded: { control: { disable: true } },
+  isBorderless: { control: { disable: true } },
```

Comment:
I disabled this for Accordion because toggling it doesn't change the blocks at all

Reply:
Did you try to make them work? I think the expandable block is just missing `{...args}` available from the story parameter.

Usually we only disable these if it's not possible to control, or if it's too complicated, or if it doesn't make sense in the story's context, or if it's adding too much noise (e.g. ReactNode props).

Reply:
This has been updated